pyiron_base/jobs/job/runfunction.py

```diff
@@ -102,6 +105,16 @@ def run_job_with_status_created(job):
         job.run_static()
     elif job.server.run_mode.srun:
         run_job_with_runmode_srun(job=job)
+    elif job.server.run_mode.executor:
+        if job.server.gpus is not None:
+            gpus_per_slot = int(job.server.gpus / job.server.cores)
```

Comment:
I'm a bit uncomfortable with the fact that there's no doc in `job.server.gpus` and also no error check there, so I might want to see a simple check like `assert gpus_per_slot > 0` at this point.

Reply:
I added a docstring to `job.server.gpus` as when as the recommended check if `gpus_per_slot > 0`.